omega/omega-transaction/src/main/java/org/apache/servicecomb/pack/omega/transaction/SagaStartAspect.java

```diff
@@ -69,6 +69,9 @@ Object advise(ProceedingJoinPoint joinPoint, SagaStart sagaStart) throws Throwab
   }
 
   private void initializeOmegaContext() {
+    if (context.globalTxId() != null) {
+      context.setParentTxId(context.globalTxId());
+    }
```

Comment:
@WillemJiang my changes is here.

Reply:
I think we should set the parentTxId from the localTxId, as the globalTxId won't be changed.

Reply:
No, I don't think so. The nested saga could be cancelled individually that means it should have the different the globalTxId.
```
SagaStart (globalTxId = 1, localTxId = 1, parentTxId = null)
Nested SagaStart (globalTxId = 2, localTxId = 1, parentTxId = 1)
Cancel Nested Saga ( this could rollback the globalTxId = 2)
Close Saga ( this could close the globalTxId = 1

Reply:
The other case is when parent saga is cancelled, it should cancel the nested saga whenever it close before or not.